geoips/plugins/modules/procflows/single_source.py

```diff
@@ -349,13 +349,15 @@ def process_xarray_dict_to_output_format(
 def print_area_def(area_def, print_str):
     """Print area def."""
     LOG.info(
-        f"\n\n************************************************************************************"
-        f"\n***{print_str}\n{area_def}"
+        """\n\n*************************************************************************
```

Comment:
Probably another one that should be separate lines 
"line1"
"line2" 
?

Reply:
fixed this